pkg/apis/scheduling/v1alpha1/register.go

```diff
@@ -42,5 +42,5 @@ func init() {
 	// We only register manually written functions here. The registration of the
 	// generated functions takes place in the generated files. The separation
 	// makes the code compile even when the generated files are missing.
-	localSchemeBuilder.Register(RegisterDefaults)
+	localSchemeBuilder.Register(addDefaultingFuncs)
```

Comment:
There doesn't seem to be an `addDefaultingFuncs` in this pkg ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/scheduling/v1beta1/defaults.go

Should there be one? This needs to be fixed for tests to pass.
/lgtm cancel

Reply:
looks like it just got deleted in https://github.com/kubernetes/kubernetes/commit/1a1b7001d6e5dbc8f2f363a39e684fb58da35ddb#diff-55191462e82a442246b6c7b9bcd928c9

Reply:
@liggitt @alejandrox1 thanks for your review. 

I add the `addDefaultingFuncs ` func into the `defaults.go` like other `apis` pkg.